webapp/channels/src/plugins/registry.ts

```diff
@@ -126,6 +126,18 @@ export default class PluginRegistry {
         return dispatchPluginComponentAction('PostMessageAttachment', this.id, component);
     });
 
+    // Register a component to the add to the post message menu.
+    // Accepts a React component. Returns a unique identifier.
+    registerPostActionComponent = reArg(['component'], ({component}: DPluginComponentProp) => {
+        return dispatchPluginComponentAction('PostAction', this.id, component);
+    });
+
+    // Register a component to the add to the post editor menu.
+    // Accepts a React component. Returns a unique identifier.
+    registerPostEditorActionComponent = reArg(['component'], ({component}: DPluginComponentProp) => {
```

Comment:
nit: `PostEditor` makes it sound like this appears where I can edit posts, but I think this only works for `PostCreator` at the present?

Reply:
It is adding the functionallity to the post advanced editor, that I would expect, eventually to be use for editing posts too. I think this name is correct.